### PR TITLE
fix: prefer canonical artist bio on artwork screen

### PR DESCRIPTION
**Pull request by anandaroop**
This PR resolves [PBRW-127] and [PBRW-249] 

### Description

On web/mWeb when we display an artist's bio within the context of an artwork page, [we prefer the canonical version of the bio](https://github.com/artsy/force/pull/7748).

But it looks like we never made the equivalent change in the app, so this addresses that.

| Before | After (iOS) | After (Android) | Web (for comparison) |
|--------|--------|--------|--------|
| before | after | android | web | 

### PR Checklist

- [x] I have tested my changes on **iOS** and **Android**.
- [x] I hid my changes behind a **[feature flag]**, or they don't need one.
- [x] I have included **screenshots** or **videos**, or I have not changed the UI.
- [x] I have added **tests**, or my changes don't require any.
- [x] I added an **[app state migration]**, or my changes do not require one.
- [x] I have documented any **follow-up work** that this PR will require, or it does not require any.
- [x] I have added a **changelog entry** below, or my changes do not require one.

### To the reviewers 👀

- [ ] I would like **at least one** of the reviewers to **run** this PR on the simulator or device.

<details><summary>Changelog updates</summary>

#### Cross-platform user-facing changes

- Prefer canonical artist bio on artwork screen

#### iOS user-facing changes

-

#### Android user-facing changes

-

#### Dev changes

-

</details>

Need help with something? Have a look at our [docs], or get in touch with us.

[app state migration]: ../blob/main/docs/adding_state_migrations.md
[feature flag]: ../blob/main/docs/developing_a_feature.md
[docs]: ../blob/main/docs/README.md

[PBRW-127]: https://artsyproduct.atlassian.net/browse/PBRW-127?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[PBRW-249]: https://artsyproduct.atlassian.net/browse/PBRW-249?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by ArtsyOpenSource** [comment]
### This PR contains the following changes:

- Cross-platform user-facing changes (Prefer canonical artist bio on artwork screen - anandaroop)
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 92f9baec4302ae072fdf9a052b6e2b6aad15a2e2
</p>

File: src/app/Scenes/Artwork/Components/AboutArtist.tsx [hunk 1]
```diff
@@ -20,6 +20,19 @@ export const AboutArtist: React.FC<AboutArtistProps> = ({ artwork }) => {
...
+    JSON.stringify(
+      {
+        hasSingleArtist,
+        biographyBlurb,
+        text,
+      },
+      null,
+      2
+    )
+  )
```
Comment (gkartalis) [inline src/app/Scenes/Artwork/Components/AboutArtist.tsx:34]: 👀👀 do we still need this?

**Review by gkartalis — DISMISSED** [review]

File: src/app/Scenes/Artwork/Components/AboutArtist.tsx (hunk 1 above)
Comment (anandaroop) [reply src/app/Scenes/Artwork/Components/AboutArtist.tsx]: 🤦🏽 fixed now

**Review by jonallured — APPROVED** [review]
LGTM!! 🎉

**Pull request merged**